src/main/java/BlackJack/CardMachine.java

```diff
@@ -0,0 +1,29 @@
+package BlackJack;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CardMachine {
+    private List<Card> cards = new ArrayList<>();
+
+    public List<CardShape> createCardShapes() {
+        return Arrays.stream(CardShape.values())
+                .collect(Collectors.toList());
+    }
+
+    public List<CardNumber> createCardNumbers() {
+        return Arrays.stream(CardNumber.values())
+                .collect(Collectors.toList());
+    }
+
+    public CardPack createCards() {
+        for (CardShape cardShape : createCardShapes()) {
+            for (CardNumber cardNumber : createCardNumbers()) {
+                cards.add(new Card(cardShape, cardNumber));
+            }
+        }
```

Comment:
배열(= XXX.values())을 List로 변환하는 작업 없이 아래와 같이 바로 Cards를 생성하는 기능을 수행해도 괜찮을 것 같아요 😃

```suggestion
        for (CardShape cardShape : CardShape.values()) {
            for (CardNumber cardNumber : CardNumber.values()) {
                cards.add(new Card(cardShape, cardNumber));
            }
        }
```